src/components/helperFunctions.js

```diff
@@ -0,0 +1,29 @@
+export const findImageSize = (image, imageSizes) => {
+  // If not both image and imageSizes are true then return empty object
+  if (!(image && imageSizes)) return null;
+
+  // If image is a path, get only the image name
+  const imageName = image.split('/').pop();
+
+  const element = imageSizes.find(
+    imageSize => imageSize.node.relativePath === imageName,
+  );
+
+  return element ? element.node.childImageSharp.sizes : null;
+};
+
+export const findImageResolution = (image, imageResolutions) => {
```

Comment:
Her finner du et sett med bilder som tilhører denne bilde-URLen. Så sett denne metoden i flertall for å forhindre forvirring.